fictionary/__init__.py

```diff
@@ -201,21 +208,45 @@ def generate_data_file(self, data_file_path, filesets):
             makedirs(containing_dir)
         self._shelf = shelve.open(
             data_file_path, protocol=2, flag='n', writeback=True)
-        self._shelf['wordlist'] = set()
-        for dictionary in ['all', 'british', 'american']:
-            print("Generating '%s' dictionary... " % dictionary,
-                  file=sys.stderr)
+        self._shelf[WORDLIST_KEY] = set()
+        for dictionary in [DICT_ALL_KEY, DICT_BRITISH_KEY, DICT_AMERICAN_KEY]:
+            LOG.debug("Started generating '%s' dictionary... " % dictionary)
             sys.stderr.flush()
             self._shelf[dictionary] = self.generate_word_list(
                 filesets[dictionary]
             )
-            print('Done.', file=sys.stderr)
+            LOG.debug("Finished generating '%s' dictionary." % dictionary)
 
     def is_real_word(self, word):
         """
         Checks the provided word to see if it's contained in the data file.
         """
-        return word in self._shelf['wordlist']
+        return word in self._shelf[WORDLIST_KEY]
+
+
+def get_random_words(
+        num_words=DEFAULT_NUM_WORDS, min_length=DEFAULT_MIN_LENGTH,
+        max_length=None, dictionary=DICT_BRITISH_KEY,
+        is_refresh=False, is_verbose=False):
+    """Get a random sequence of fictionary words.
+
+    Call this function to use fictionary from any other Python code.
+    """
+    LOG.setLevel(logging.DEBUG if is_verbose else logging.WARNING)
```

Comment:
This should be done in the `main` function, and then the `is_verbose` parameter can be removed from this function. Setting logging verbosity is a function of the command-line app, not something to be done in a utility function.

Reply:
Ok, done.